src/models.ts

```diff
@@ -104,8 +105,23 @@ export interface IPage {
   displayName: string;
 }
 
+export interface IVisual {
```

Comment:
we probably don't have to add IVisual now since we didn't fully decide if we go with ids or tags?